frontend/src/components/pages/HomePage/LogRecordsTable.tsx

```diff
@@ -85,6 +85,31 @@ const LogRecordsTable = ({
     }));
   };
 
+  const formatNote = (note: string) => {
+    const noteLimit = 150;
```

Comment:
nit: Move this up higher in the file (reduces memory of function invocation) + renaming it to adhere to other constant names

```suggestion
    const NOTE_LIMIT = 150;
```

Reply:
I will make the change but just for my own learning how does moving the function reduce memory usage?

Reply:
@owen-sellner basically everytime you invoke this function you're declaring this as a new variable - since it never changes you can put it as a constant at the top of the file so it's only declared once